### DISCUSSION TITLE
Allow dynamic assignment of task ownership

### DISCUSSION THREAD

**Discussion by kagoodm** [SalesforceLabs/ActionPlansV4#164] [Ideas]
It would be great if we could add a field onto action plan template task to identify a specific field from your reference record that you want to use as the task owner upon creation.

Example:   On our opportunity object, we have lookup fields against the User object for account manager and project manager.  On our task plan template, some of the tasks should be owned by the opportunity owner, some by the account manager, and some by the project manager.  If we could identify on the template which reference field to use and actually have it incorporated into the code to use that field, that would be so helpful.

**Comment by jsams-merkle** [comment]
Bonus points if we were able to use Roles from the Opportunity Team to dynamically assign AP Task ownership. Our org uses Opportunity Team and Account Team because we don't want to have a bunch of user lookup fields on these objects and because what roles need to be specified vary based on different, sometimes subjective, criteria. If we were able to use lookup fields on the Opportunity _AND/OR_ Opportunity Team roles, that would be most helpful.
